zeton/templates/index_child.html

```diff
@@ -21,67 +21,66 @@ <h2 style="color: green;">Możesz korzystać z komputera</h2>
             <h2 class="text-center">Sposób na zdobycie punktów</h2>
             <br>
             <ul>
-            <table>
-                <tr>
-                    <th>Id</th>
-                    <th>Nazwa</th>
-                    <th>Punkty</th>
-                    <th>Max. dziennie</th>
-                    <th>Max. tygodniowo</th>
-                </tr>
-                {% for task in childs_tasks %}
-                <tr>
-                    <td>{{ loop.index }}</td>
-                    <td>{{ task.name }}</td>
-                    <td>{{ task.points }}</td>
-                    <td>{{ task.max_day }}</td>
-                    <td>{{ task.max_week }}</td>
-                </tr>
-                {% endfor %}
-            </table>
+                <table>
+                    <tr>
+                        <th>Id</th>
+                        <th>Nazwa</th>
+                        <th>Punkty</th>
+                        <th>Max. dziennie</th>
+                        <th>Max. tygodniowo</th>
+                    </tr>
+                    {% for task in childs_tasks %}
+                    <tr>
+                        <td>{{ loop.index }}</td>
+                        <td>{{ task.name }}</td>
+                        <td>{{ task.points }}</td>
+                        <td>{{ task.max_day }}</td>
+                        <td>{{ task.max_week }}</td>
+                    </tr>
+                    {% endfor %}
+                </table>
             </ul>
 
             <h5 class="text-center">SZKOLNY REKORD TYGODNIA--> chipsy & extra 100 pkt</h5>
         </div>
 
         <div id="odejmowanie" class="col-xl-6 square">
+            <form>
+                <div class="form-group button">
+                    <a class="btn btn-info" href="/prizes_detail/{{ child.id }}">Szczegóły</a>
+                </div>
+            </form>
             <h2 class="text-center">Wykorzystaj punkty</h2>
 
             {% if role == 'caregiver' %}
             <form action="/api/child/{{ child.id }}/points/use?return_url=/child/{{ child.id }}" method="post">
-            {% else %}
-            <form action="/api/child/{{ child.id }}/points/use" method="post">
-            {% endif %}
-                <div class="form-group button">
-                    Liczba punktów: <input class="form-control" type="number" name="points" required>
+                {% else %}
+                <form action="/api/child/{{ child.id }}/points/use" method="post">
```

Comment:
zduplikowany kod/ prawdopodobnie powstał wcześniej. Teraz jest dobra okazja do jego usunięcia. W tym miejscu nie potrzebnie jest sprawdzane. `{% if role == 'caregiver' %}`

Reply:
w myśl, że zmieniamy zasady (dziecko może wykorzystywać punkty) usuwam ten warunek

Reply:
> tu również: Chciałbym żeby były 3 przyciski strona główna, wyloguj, wróć. Sama zmiana nazwy nie wystarczy. W momencie gdy jesteśmy zalogowani jako opiekun przycisk strona główna przenosi na listę podopiecznych. Potrzebny jest dodatkowy przycisk wróc. W późniejszym czasie zrobimu porządek z tymi przyciskami przez navbar

Okay, zaraz je ogarnę

Reply:
> w myśl, że zmieniamy zasady (dziecko może wykorzystywać punkty) usuwam ten warunek

Zwracam honor, zakręciłam się :)

Reply:
> zduplikowany kod/ prawdopodobnie powstał wcześniej. Teraz jest dobra okazja do jego usunięcia. W tym miejscu nie potrzebnie jest sprawdzane. `{% if role == 'caregiver' %}`

To odnośnie pliku index_child.html. 

Wiesz co, skasowałam ten kod i pojawił się błąd z widoku dziecka, dziecko nie mogło wykorzystywać punktów ("Forbidden You don't have the permission to access the requested resource. It is either read-protected or not readable by the server.").

Po wykorzystaniu punktów inny url path jest dla opiekuna, a inny dla dziecka.
Wydaje mi się, że ten kod musi pozostać taki jaki jest?

No nie wiem, pomóżcie, bo może coś źle robię :(

Reply:
`index_child.html` zostawiamy  - bez tego kodu faktycznie nie działa poprawnie. sorki za zamieszanie.